File: Automation/001-AutomationFramework/C#/AutomationFramework/Engines/SeleniumEngine/SeleniumManager.cs
using CommonCore.Contracts;
using CommonCore.Tests;
using SeleniumEngine.DriverInitialization;

namespace SeleniumEngine
{
    public class SeleniumManager : IEngineManager
    {
        public void StartUp()
        {
            TestInMemoryParameters.Instance.DriverType = "Edge";

            WebDriverFactory.InitializeWebDriver(TestInMemoryParameters.Instance.DriverType);
            //featureContext["WebDriver"] = WebDriverFactory.WebDriver;
            //ContainerManager.RegisterType(singleCapability.Name, driver);

        }

        public void ShutDown()
        {
            WebDriverFactory.CleanUp();
        }

        public string CollectData(string destinationPath, string eventName)
        {
            throw new NotImplementedException();
        }
    }
}

File: Automation/Woolsocks/TestAutomationFramework/src/main/java/engines/selenium/SeleniumManager.java
package engines.selenium;

import common.core.contracts.IEngineManager;
import engines.selenium.driverInitialization.WebDriverFactory;

public class SeleniumManager implements IEngineManager {
    public void startUp() {
        try {
            WebDriverFactory.getThreadSafeInstance().initializeWebDriver(30,30);
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }

    public void shutDown() {
        WebDriverFactory.getThreadSafeInstance().cleanUp();
    }

    public String collectData(String destinationPath, String eventName) {

        return "";
    }
}
